PIL/Image.py

```diff
@@ -28,8 +28,11 @@
 
 from PIL import VERSION, PILLOW_VERSION, _plugins
 
+import logging
 import warnings
 
+logger = logging.getLogger(__name__)
```

Comment:
I suppose we should always use name 'pillow' (or something similar) for all logging from the library. This will allow to easily manage logging from external code.